src/helpers/flow-chart.ts

```diff
@@ -33,6 +33,18 @@ export const formatForFlowchart = (data: ParsedYAML) => {
 
   let prevNode = false;
 
+  if (Array.isArray(pods)) {
+    let newPods = {};
```

Comment:
I would make it more verbose. Like 

switch(data.version) {
     case "1"
}

This would communicate to future developers why we are handling the case if pods are in an array